transformations/country_state_abbreviation_transformation/README.md

```diff
@@ -25,6 +25,21 @@ This transformation provides flexible options for Country/State abbreviation and
 | abbr  |  Boolean | Enable/disable full name -> abbreviation  |  True  | 
 | exp  | Boolean  | Enable/disable abbreviation -> full name | True  |  
 
+Benchmark results:
+
+- Sentiment analysis: 
+```
+python evaluate.py -t CountryStateAbbreviation -task TEXT_CLASSIFICATION
+```
+The original accuracy is 96.0 and the perturbed accuracy is 96.0.
+
+- Text summarization: 
+```
+python evaluate.py -t CountryStateAbbreviation -task TEXT_TO_TEXT_GENERATION
+```
+The original bleu is 14.88 and the perturbed bleu is 14.61.
+
+
```

Comment:
Just to confirm, have you used the models used [here](https://github.com/GEM-benchmark/NL-Augmenter/tree/main/evaluation) i.e. from textattack ?

Reply:
TEXT_CLASSIFICATION used model aychang/roberta-base-imdb on the test[:20%] split of the imdb dataset
TEXT_TO_TEXT_GENERATION used the model sshleifer/distilbart-xsum-12-6 on the test[:20%] split of the xsum dataset.

I can update the test classification performance with textattack models if needed. Thank you!